### ISSUE TITLE
[BUG] Incorrect focus order on sidebar viewport

### ISSUE THREAD

**Issue by xelaint** [graycoreio/daffodil#3314]
# :bug: Bug report

## Current Behavior
Incorrect focus order on sidebar viewport. Current focus order is: sidebar, nav, content.

## Expected Behavior
The focus order should be: nav, sidebar, content.

## Environment

<pre><code>
Daffodil version: X.Y.Z
Angular version: X.Y.Z 

 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>
